app/models/site_config.rb

```diff
@@ -51,6 +51,7 @@ class SiteConfig < RailsSettings::Base
   field :rate_limit_published_article_creation, type: :integer, default: 9
   field :rate_limit_image_upload, type: :integer, default: 9
   field :rate_limit_email_recipient, type: :integer, default: 5
+  field :rate_limit_article_update, type: :integer, default: 50
```

Comment:
I'd say the default here should probably be higher since 50's probably not so out of the realm of normal use for a day.

Not a blocking comment if we want to rush this.

Reply:
Updated to 150